ansible/ansible_collections/packaging/tools/tests/unit/plugins/modules/test_packaging_modules.py

```
@@ -0,0 +1,363 @@
+    )
+
+    assert_is(result["changed"], True, "uv_tool should report uninstall change")
+    assert_equal(
+        result["cmd"],
+        ["/usr/bin/uv", "tool", "uninstall", "ruff"],
+        "uv_tool should build uninstall command",
+    )
+
+
+def test_uv_tool_absent_is_idempotent_when_tool_missing(monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.setattr(uv_tool, "resolve_binary", lambda module, value: "/usr/bin/uv")
```

Ruff (0.15.11)

[warning] Unused lambda argument: `module`

(ARG005)